src/pages/_app.tsx

```diff
@@ -35,7 +36,16 @@ export default function App({ Component, pageProps }: AppProps) {
  *
  * `_app`에서만 사용되는 컴포넌트이기에, 인라인으로 작성되었습니다.
  */
+let vh = 0;
+
 function Layout({ children }: PropsWithChildren<{}>) {
+  const windowSize = useWindowSize();
+
+  useEffect(() => {
+    vh = window.innerHeight * 0.01;
+    document && document.documentElement.style.setProperty('--vh', `${vh}px`);
```

Comment:
다시 보다가 생각난건데 `effect` 내부이니 document 여부 확인 안해도 될 거같아요!!
```suggestion
    document.documentElement.style.setProperty('--vh', `${vh}px`);
```

Reply:
아하 저도 원래 `document &&` 체크를 안했었는데요, 배포했을 때 에러 발생으로 제대로 적용되지 않는 문제가 있었거든요! 그래서 경험 상 추가한 부분인데 ㅎㅎ 일단 한번 없애고 배포를 진행해볼게요! 문제 생기면 document && 다시 넣는 거 어렵지 않으니!